src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public Excavator(String machineType) {
+        super(machineType);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("Exavator" + " " + machineType + " " + "started its work.");

```

Comment:
There's a typo in the message printed to the console. It should be 'Excavator' instead of 'Exavator'. Make sure the messages are informative and correctly spelled.